src/Sylius/Bundle/CoreBundle/Fixture/Factory/OrderExampleFactory.php

```diff
@@ -129,6 +133,9 @@ public function create(array $options = []): OrderInterface
 
         $order = $this->createOrder($options['channel'], $options['customer'], $options['country'], $options['complete_date']);
         $this->setOrderCompletedDate($order, $options['complete_date']);
+        if ($options['fulfilled'] ?? false) {
```

Comment:
```suggestion
        if ($options['fulfilled']) {
```
should be enough I guess